src/searcher.cpp

```diff
@@ -192,7 +213,16 @@ inline void Searcher::add_lit_to_learnt(
     }
     seen[var] = 1;
 
+    const uint32_t lit_ind = (~lit).toInt();
```

Comment:
This is not used, remove please.

Reply:
lit_ind has been used in line 218.

Reply:
Please save the literal instead of saving the uint32_t form of it. Then covert it later. You don't need the uint32_t, you need the literal. Someone looking at the code will not understand why you convert it here and not later. Conceptually, you are interested in the literal, not its uint32_t form. 3 years later you will look at this code and will be really upset at yourself for making your own life harder. Please fix!